### PR TITLE
added a button, show 2nd page and javascript

### PR DESCRIPTION
**Pull request by AdamTorvikStaffen**

### CODE REVIEW

File: nba-insider/src/main/resources/static/index.html [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
+  <body>
+    <div class="flex-container" id="page-1">
+      <div>
+        <img class="logo" src="png/warriors_w_transparent.png" alt="Warriors logo">
+      </div>
+      <div class="title">
+        trades-R-US
+      </div>
+      <div class="button-wrapper">
+        <button onclick="myFunction()" class="generate" type="button">Generate Tradez</button>
```
Comment (Dragoris) [inline nba-insider/src/main/resources/static/index.html:20]: myFunction() is a trash function name. Be more semantic.

File: nba-insider/src/main/resources/static/myscripts.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+function myFunction() {
+  alert("Fucka you!");
```
Comment (bluecadmium) [inline nba-insider/src/main/resources/static/myscripts.js:2]: 😠  Fuckaaaaaa UUUUUU

Comment (Dragoris) [inline nba-insider/src/main/resources/static/myscripts.js:2]: remove alert and log statements

File: nba-insider/src/main/resources/static/mystyle.css [hunk 3]
```diff
@@ -1,14 +1,49 @@
...
-  left: 50%;  
+.title {
+  color: #FF6700;
+  font-family: Cooper Std Black;
+  font-size: 50px;  
+}
+.flex-container {
+  display: flex;
+  flex-wrap: nowrap;
+  background-color: DodgerBlue;
```
Comment (bluecadmium) [inline nba-insider/src/main/resources/static/mystyle.css:19]: Team Blue!

**Review by bluecadmium — APPROVED** [review]
Ship it!

File: nba-insider/src/main/resources/static/mystyle.css [hunk 4]
```diff
@@ -1,14 +1,49 @@
...
+  flex-direction: column;
+  justify-content: space-evenly;
+  height: 100vh;
+  align-items: center;
+}
+.button-wrapper {
```
Comment (Dragoris) [inline nba-insider/src/main/resources/static/mystyle.css:25]: delete empty css classes

File: nba-insider/src/main/resources/static/mystyle.css [hunk 5]
```diff
@@ -1,14 +1,49 @@
...
+
+}
+.hidden {
+  display: none;
+}
+
+.generate{
+  display:inline-block;
```
Comment (Dragoris) [inline nba-insider/src/main/resources/static/mystyle.css:33]: NIT put spaces after :

**Pull request closed without merging**